app/src/features/surveys/CreateSurveyPage.tsx

```diff
@@ -377,6 +379,13 @@ const CreateSurveyPage = () => {
 
                 <Divider className={classes.sectionDivider} />
 
+                <HorizontalSplitFormComponent
+                  title="Block Data"
+                  summary="Data for Blocks"
+                  component={<SurveyBlockSection name="blocks" />}
```

Comment:
Is the `name` prop necessary? It's the same value for both instances of `<SurveyBlockSection />`

Reply:
No not really. I think just a left over from an earlier iteration